.github/workflows/autoupdate-preview.yaml

```diff
@@ -56,9 +60,11 @@ jobs:
             ## Using early preview changes
 
             ```sh
-            go get go.mongodb.org/atlas-sdk/v20240805005@dev-latest   
+            go get go.mongodb.org/atlas-sdk/preview@preview-latest   
```

Comment:
If we merge this change, would SDK preview be usable? 
Do you intend to publish preview-latest?

Reply:
that change is outdated, it's working with latest changes